### ISSUE TITLE
This doesn't work on iOS 9 for Facebook

### ISSUE THREAD

**Issue by SMJ93** [philmerrell/CanOpen#4]
-canOpenURL: failed for URL: "fb://" - error: "This app is not allowed to query for scheme fb"
-canOpenURL: failed for URL: "fb://profile?id=********" - error: "This app is not allowed to query for scheme fb"

It works fine on <= iOS8.

**Comment by incentfit-product** [comment]
iOS 9 requires a whitelist - read this.
I'm not sure how to declare this in the PhoneGap config.xml file yet, but please post a solution here if you or anyone else figures this out.
http://awkwardhare.com/post/121196006730/quick-take-on-ios-9-url-scheme-changes

**Comment by SMJ93** [comment]
Thank you @incentfit. Sorted now :)

I used Facebook develop's documentation: https://developers.facebook.com/docs/ios/ios9

**Comment by incentfit-product** [comment]
@SMJ93 Can you paste your PhoneGap config.xml file? I'd be interested exactly how you have it configured for this.

**Comment by SMJ93** [comment]
@incentfit

I didn't touch the config.xml file.

I opened up the project in Xcode. 

Then clicked the project -> Target -> Info.plist

Under Custom iOS Target Properties I added the LSApplicationQueriesSchemes Key as an array type.

In here I inserted all of the Facebook values mentioned int eh documentation.

Like this: http://stackoverflow.com/questions/31986475/fbauth2-is-missing-from-your-info-plist-under-lsapplicationqueriesschemes-and-is
